wg-meetings/2020-02-12.md

```diff
@@ -0,0 +1,77 @@
+# Node.js Foundation Diagnostics WorkGroup Meeting 2020-02-12
+
+## Links
+
+* **Recording**:  
+* **GitHub Issue**: $GITHUB_ISSUE$
+* **Minutes Google Doc**: $MINUTES_DOC$
+
+## Present
+
+* Diagnostics team: @nodejs/diagnostics
+* David Anderson (@daviande)
+* Matheus Marchini (@mmarchini)
+* Michael Dawson (@mhdawson)
+* Peter Marton (@hekike)
+* Ruben Bridgewater (@BridgeAR)
+
+## Agenda
+
+## Announcements
+ 
+*Extracted from **diag-agenda** labelled issues and pull requests from the
+**nodejs org** prior to the meeting.
+
+### nodejs/diagnostics
+
+* Future of llnode [#355](https://github.com/nodejs/diagnostics/issues/355)
+ * Mattheus: proposing to remove review restrictions because other maintainers 
+   are less active and this is blocking to make progress on important things
+   like Node 12.x release. This one took months and we fell behind the LTS
+   schedule. Which prevented people from using llnode on Node 12.x. Because of
+   the nature of llnode it needs rabid iteration to keep it sync with V8
+   upgrades.
+ * Michael: we could introduce something like after some time it can land if
+   nobody looks at it.
+ * Matheus: that could work
+ * Michael: what would be the reasonable time for this auto approval window?
+ * Mattheus: how about 3-4 days?
```

Comment:
How about 7 days, since that is the amount of time required in `nodejs/node` with a single approval.